src/Microsoft.ApplicationInsights.AspNet/JavaScript/IApplicationInsightsJavaScript.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace Microsoft.ApplicationInsights.AspNet.JavaScript
+{
+    public interface IJavaScriptSnippet
```

Comment:
Oh, I see. I have hard time imagining someone doing that though. It could also be done without the interface. 

We should talk to @MaxShehovtsov and @scsouthw before deciding to put this in the Microsoft.ApplicationInsights.JavaScript NuGet package. I could also imagine it being used in non-ASP.NET project types, like Cordova and WWA.